crates/sui-indexer-alt-jsonrpc/src/data/object_versions.rs

```diff
@@ -37,13 +37,17 @@ impl Loader<LatestObjectVersionKey> for PgReader {
             r#"
                 SELECT
                     k.object_id,
-                    v.object_version
+                    v.object_version,
+                    v.object_digest,
```

Comment:
`obj_versions` doesn't have an index that has `object_digest`. So does that mean this query will fetch actual data pages instead of just index. And how does that affect the performance of this query?

Reply:
From the PR description:

> This slightly undoes an optimisation that was introduced for LatestObjectVersionKey, where it was optimised to only request columns that were in the index being scanned (so the fetch could be an index only scan). I have confirmed that after switching to `CROSS LATERAL JOIN` this doesn't hurt performance (I suspect this is because when we were using `SELECT DISTINCT ON`, it was loading data before filtering, to sort, and it can avoid doing that entirely when we use `CROSS JOIN LATERAL` so the number of heap pages it needs to fetch is now much smaller).